src/components/EditorOperatorArgument.vue

```diff
@@ -77,21 +77,28 @@
         </div>
       </div>
     </div>
-    <div v-if="argument.markupType === 'subscript'" class>
-      Subscipts are not supported yet.
+    <div v-if="argument.markupType === 'script'" class="subscripts">
+      <p v-if="!subscript" class="empty">
```

Comment:
I think we can't have `argument.markupType === 'script'` and `!subscript` at the same time. You can delete empty case branch.